Gallery modal, optimizations, schedule changes

smartspot2 opened the pull request:
- Added a modal upon click of a gallery image with a larger version of the image
- Resized the gallery images to optimize page load time, changed gallery images to better ones (#10)
- Added prefixes to several CSS properties to ensure multiple browser support (#20)
- Revised schedule script to make it more readable (from #6)
- ~~Left panel of schedule now updates upon a click of an event in the right panel~~
- Schedule design changed:
  - Removed left panel, put schedules side by side
  - Reorganized schedule data structure
  - Popover dialog box on click of event showing extra details

smartspot2 committed:
Added CSS properties for multiple browser support

smartspot2 committed:
Resized images, added gallery modal

Changed out the gallery images to better ones and downsized them.
Added larger versions of gallery images to appear on click in a modal.

smartspot2 committed:
Reorganized schedule script, added click event functionality

Optimized some code, reduced repetitions of lines.
When user clicks on event, the active event shown on the left panel now updates to match.

smartspot2 committed:
Updated image alt tags

alexnovo52 commented:
Good! Can you add next picture and before picture buttons for the gallery? It feels a bit tedious to click back every time to go to the next picture. Also I recommend you convert the pictures to WebP. Its a new file format by Google emphasizing size and efficiency on websites.

smartspot2 committed:
Added gallery modal prev/next buttons

Also changed the disable scrolling behavior to include all future modals as well

smartspot2 committed:
Changed schedule design

Removed left panel, put schedules side by side.
Added detail popover on click of event.
Reorganized schedule data structure as a JS object, dynamically creating the schedule in HTML.
